photon-ml/src/main/scala/com/linkedin/photon/ml/util/Logging.scala

```diff
@@ -25,13 +25,13 @@ import org.slf4j.{Logger, LoggerFactory}
  * https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/Logging.scala
  */
 protected[ml] trait Logging {
-  // Transient so that it can be used distributed code without incurring unecessary serialization / io
-  @transient private var log: Logger = null
+  // Transient so that it can be used distributed code without incurring unnecessary serialization / io
+  @transient private var log: Logger = _
```

Comment:
Let's leave this as "null". I know it's not idiomatic, but I kept it this way on purpose to more clearly match the condition on L43.